### DISCUSSION TITLE
LTS Licensing

### DISCUSSION THREAD

**Discussion by TIncorviaITLS** [primefaces/community#1039] [PrimeNG]
In the PrimeNG LTS FAQ, it is stated that the duration of the LTS Extended license is unlimited.  Please clarify: if I am using, for instance, v16 under the LTS license, does “unlimited” mean that I will get long term support for v16 in perpetuity?  I.e., forwver?

**Comment by cetincakiroglu** [comment]
Hi,

Could you please send an email to contact@primetek.com.tr

**Reply by TIncorviaITLS** [reply]
Please reply in the discussion thread as this is a question that will be important to all who use PrimeNG.  Much appreciated.
